files/system/wayfire/etc/environment

```diff
@@ -1,3 +0,0 @@
-WAYFIRE_CONFIG_FILE=/usr/share/wayfire/wayfire.ini
```

Comment:
is the default theme still adwaita-dark with this removed?

Reply:
no its not sorry for being careless